teams/teams-ps/teams/Set-CsTenantFederationConfiguration.md

```diff
@@ -167,6 +168,14 @@ When the BlockAllSubdomains is enabled, all users from all subdomains of all dom
 So, users from subdomain.contoso.com and subdomain.fabrikam.com will be blocked.
 Note: Users from subcontoso.com will not be blocked because it's a completely different domain rather than a subdomain of contoso.com.
 
+### -------------------------- Example 11 -------------------------
+```
+Set-CsTenantFederationConfiguration -ExternalAccessWithTrialTenants "Allowed"
+```
+
+Example 11 shows how the external access to be able to communicate with tenants that contain trial users can be allowed (default value is Blocked).
```

Comment:
"Example 11 shows how the external access to be able to communicate with tenants that contain trial users can be allowed (default value is Blocked)."

should say the following to align with the text used in examples 10 and others:

"Example 11 shows how you can allow users to communicate with users in tenants that contain only trial users (default value is Blocked)."